bears/general/InvalidLinkBear.py

```diff
@@ -57,24 +57,64 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             [^.:%\s_/?#[\]@\\]+         # Initial part of domain
             \.                          # A required dot `.`
             (
-                (?:[^\s()%\'"`<>|\\]+)  # Path name
+                (?:[^\s()%\'\"`<>|\\]+) # Path name
                                         # This part does not allow
                                         # any parenthesis: balanced or
                                         # unbalanced.
             |                           # OR
-                \([^\s()%\'"`<>|\\]*\)  # Path name contained within ()
+                \([^\s()%\'\"`<>|\\]*\) # Path name contained within ()
```

Comment:
```
Python 3.5.2 (default, Sep 14 2016, 11:28:32) 
[GCC 6.2.1 20160901 (Red Hat 6.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> re.compile('"')
re.compile('"')
>>> 
```
no errors. I think this is because regex101 assumes that you wrap your regex string inside double quotes.